modcp.php

```diff
@@ -2452,6 +2460,10 @@
 			$attachment['filename'] = htmlspecialchars_uni($attachment['filename']);
 			$attachment['threadsubject'] = htmlspecialchars_uni($parser->parse_badwords($attachment['threadsubject']));
 			$attachment['filesize'] = get_friendly_size($attachment['filesize']);
+			if(empty($attachment['postsubject']))
+			{
+				$attachment['postsubject'] = "RE: ".$attachment['threadsubject'];
```

Comment:
This should probably be translatable via a language variable.

Reply:
The `RE:` prefix for thread replies is indeed hardcoded deeply in current MyBB.

I'll only try to update those variables in this PR with a translatable entry.

Reply:
I feared that may be the case. In that case, just leave the ones in this PR for now, and we'll open another issue to try and patch them all at once.